components/brave_rewards_ui/components/settings.js

```diff
@@ -0,0 +1,336 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
how is this file used? I don't see it added to a gn file anywhere

Reply:
I think this is probably supposed to go in resources/BUILD.gn

Reply:
this is called from components/brave_rewards_ui/components/app.js

Reply:
then it needs to be added to resources/BUILD.gn or changes will not trigger webpack to rebuild

Reply:
every input file should always be added to `resources/BUILD.gn` even if it is included by another file because the build system doesn't understand js

Reply:
ok sounds good will add it

Reply:
I think it's not needed, because every brave_*_ui folder provides only one file via webpack. Let me know if it's still needed